src/accountsdb/db.zig

```diff
@@ -1548,8 +1610,19 @@ pub const AccountsDB = struct {
                 }
             }
             references_to_delete.clearRetainingCapacity();
+            self.stats.clean_references_deleted.set(references_to_delete.items.len);
+            self.logger.debugf(
+                "cleaned slot {} -  old_state: {}, zero_lamports: {}",
+                .{ account_file.slot, num_old_states, num_zero_lamports },
+            );
         }
 
+        self.stats.clean_files_queued_deletion.set(delete_account_files.count());
+        self.stats.clean_files_queued_shrink.set(delete_account_files.count());
+        self.stats.clean_slot_old_state.set(num_old_states);
+        self.stats.clean_slot_zero_lamports.set(num_zero_lamports);
+
+        self.logger.debugf("cleanAccountFiles time elapsed: {}ns", .{timer.read()});
```

Comment:
should probably move elapsed times into a bucket -- see how we handle it in gossip service `processMessages`